codegen/src/main/java/io/improbable/keanu/codegen/python/ParamStringProcessor.java

```diff
@@ -0,0 +1,24 @@
+package io.improbable.keanu.codegen.python;
+
+import com.google.common.collect.ImmutableMap;
+import com.sun.javadoc.ConstructorDoc;
+import com.sun.javadoc.Tag;
+
+import java.util.Map;
+
+import static com.google.common.base.CaseFormat.LOWER_UNDERSCORE;
+import static com.google.common.base.CaseFormat.UPPER_CAMEL;
+
+class ParamStringProcessor {
+    static Map<String, String> getNameToCommentMapping(ConstructorDoc constructorDoc) {
+        ImmutableMap.Builder<String, String> nameToCommentMapping = ImmutableMap.builder();
+        Tag[] params = constructorDoc.tags("@param");
+        for (Tag param: params) {
+            String[] text = param.text().split(" ", 2);
+            String snakeCaseParamName = UPPER_CAMEL.to(LOWER_UNDERSCORE, text[0]);
+            String paramComment = text[1].trim();
```

Comment:
Can you pull out the `text[1]` into a variable so it's a bit clearer? Is text[0] the name and text[1] the description?